src/pipelines/sentiment.rs

```diff
@@ -141,8 +142,10 @@ impl SentimentModel {
         let labels = self.sequence_classification_model.predict(input);
         let mut sentiments = Vec::with_capacity(labels.len());
         for label in labels {
-            let polarity = if label.id == 1 {
+            let polarity = if label.id == 4 || label.id == 3 {
```

Comment:
Okay so this basically gains us sst5 but loses us sst2 but that's fine because the _current_ production model we're using is sst5. And this is why, as you mention in the PR description, it'd be great to get this into `rust-bert` as something configurable per model so the sentiment pipeline works with whatever IDs the underlying model will return?

Reply:
Yes - and I think it would be a fun rust-bert contribution to try, but maybe tricky.

Reply:
Yeah I wonder how these models work and if we could get "lucky" with a "specify the neutral range and anything above that is positive and anything below is negative". But it's technically more flexible to require the mapping as you say. Yeah interesting to try and figure out.

Reply:
Well, the models often come with something like `config.json` which explicitly contains the mapping.